# Boilerplate ---------------------------------
boilerplate:
  joint_state_topic: /ROBOT/joint_states
  arm_joint_model_group: arm

# Execution Interface ------------------------
execution_interface:
  command_mode: joint_execution_manager
  cartesian_command_topic: /execution_interface/cartesian_command
  joint_trajectory_topic: /ROBOT/position_trajectory_controller/command
  save_traj_to_file: false
  save_traj_to_file_path: ~/ROBOT_trajectory_data/
  visualize_trajectory_line: false
  visualize_trajectory_path: false
  check_for_waypoint_jumps: false
